scripts/output_env.sh

```diff
@@ -23,11 +23,15 @@ print_version()
     shift
     ARGS="$1"
     shift
-    FAIL_MSG="$1"
+    VARIANT=$1
     shift
 
+    if [ ! -z $VARIANT ]; then
```

Comment:
```suggestion
    if [ -n "$VARIANT" ]; then
```
You need double quotes, otherwise the shell expands wildcards in the value of the variable and splits it into words.

`! -z` is a convoluted way of writing `-n`.